build.sbt

```diff
@@ -4,15 +4,17 @@ description := "Scala syntax for Guice"
 
 organization := "net.codingwell"
 
-version := "4.2.6"
+version := "4.2.7"
 
 licenses := Seq("Apache 2" -> new URL("http://www.apache.org/licenses/LICENSE-2.0.txt"))
 
 homepage := Some(url("https://github.com/codingwell/scala-guice"))
 
 libraryDependencies ++= Seq(
-  "com.google.inject" % "guice" % "4.2.2",
-  "com.google.guava" % "guava" % "25.1-android",

```

Comment:
dropped this and tests are still fine.
i checked `dependecyList` on all scala versions.
for 2.11 its guava is still there as 25.1-android
for 2.12 and 2.13 guava is there as 27.1-jre